py/desimodel/test/test_healpix.py

```diff
@@ -18,10 +18,10 @@ def test_tiles2pix(self):
         tiles = desimodel.io.load_tiles()
 
         pix = tiles2pix(nside=8, tiles=tiles[:3], radius=1.6)
-        self.assertTrue( np.all( (pix, [196,197,208,302,303]) ) )
```

Comment:
I originally used this notation
```
self.assertTrue( np.all(pix == [196,197,208,302,303]) )
```
but got a deprecation warning (and I thought this might be the reason tests were failing.  Do I believe this is preferred --
```
self.assertTrue( np.all( (pix, [196,197,208,302,303]) ) )		
```
So you may choose to revert this change here and below.